src/map/pc.h

```diff
@@ -888,7 +904,7 @@ END_ZEROED_BLOCK; /* End */
 	int (*dropitem) (struct map_session_data *sd,int n,int amount);
 
 	bool (*isequipped) (struct map_session_data *sd, int nameid);
-	bool (*can_Adopt) (struct map_session_data *p1_sd, struct map_session_data *p2_sd, struct map_session_data *b_sd);
+	enum adopt_responses (*try_adopt) (struct map_session_data *p1_sd, struct map_session_data *p2_sd, struct map_session_data *b_sd);
```

Comment:
Note for me later: this requires an edit to the HPM Hooks generator script so that it'll know what the "default" return value should be